test.js

```diff
@@ -51,6 +51,56 @@ Ponies|
 	t.is(actual, expected);
 });
 
+test('main2', t => {
```

Comment:
This needs a better name.

Reply:
Renamed to `test_quotes_encoding_in_href`